varnishgather

```diff
@@ -393,6 +401,10 @@ _EOF_
 	exit $1
 }
 
+suggest_n_values() {
```

Comment:
this looks like an unnecessary level of indirection? If you want to use `get_varnishd_instances`, just use `get_varnishd_instances`